trio/testing/_memory_streams.py

```diff
@@ -1,7 +1,7 @@
 from __future__ import annotations
 
 import operator
-from typing import TYPE_CHECKING, Awaitable, Callable
+from typing import TYPE_CHECKING, Awaitable, Callable, TypeVar
```

Comment:
I would do `from collections.abc import Awaitable, Callable`

Reply:
In this case it's necessary because of the type aliases below.

Reply:
That kind of sucks but I understand